sdk/python/jobs/automl-standalone-jobs/automl-image-instance-segmentation-task-fridge-items/automl-image-instance-segmentation-task-fridge-items.ipynb

```diff
@@ -537,7 +537,7 @@
     "    compute_config = AmlCompute(\n",
     "        name=compute_name,\n",
     "        type=\"amlcompute\",\n",
-    "        size=\"Standard_NC6\",\n",
+    "        size=\"Standard_NC6s_v3\",\n",
```

Comment:
how about the v2 notebook for OD?

Reply:
OD has already updated with nc6sv3 https://github.com/Azure/azureml-examples/blob/main/sdk/python/jobs/automl-standalone-jobs/automl-image-object-detection-task-fridge-items/automl-image-object-detection-task-fridge-items.ipynb